src/workerd/server/gpu.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace workerd::gpu {
```

Comment:
Non-blocking nit: for someone who may stumble upon this, it's likely worthwhile adding a comment that this is currently experimental and WIP.